script.js

```diff
@@ -0,0 +1,64 @@
+// Checking if local storage is empty then add an empty array
+if (localStorage.getItem('Added Books') == null) {
+  localStorage.setItem('Added Books', JSON.stringify([]));
+}
+
+// store data into local storage
+const storeData = JSON.parse(localStorage.getItem('Added Books'));
+
+function updateData() {
+  localStorage.setItem('Added Books', JSON.stringify(storeData));
+}
+
+function createBooks(arr) {
+  let books = '';
+  for (let i = 0; i < arr.length; i += 1) {
+    books += `
+          ${arr[i].title}<br>
+          ${arr[i].author}<br>
+          <button onclick="removeBook('${i}')">Remove</button>
+          <hr/>
+          `;
+  }
+  return books;
+}
+
+// Displaying data to the UI from local storage
+function displayBooks() {
+  const listOfBooks = document.querySelector('.container');
+  listOfBooks.innerHTML = `
+         <p>
+          ${createBooks(storeData)}
+          </p>
+  `;
+}
+
+// Adding new data in the local storage
+function addNewdata(bookTitle, bookAuthor) {
+  const Book = {
+    title: bookTitle,
+    author: bookAuthor,
+  };
+  storeData.push(Book);
+  updateData();
+  displayBooks();
+}
+
+// Getting values from input fields
+const form = document.querySelector('form');
+form.addEventListener('submit', (e) => {
+  const title = document.querySelector('.title');
+  const author = document.querySelector('.author');
+  e.preventDefault();
+  addNewdata(title.value, author.value);
+});
+
+// Removing data from local storage
+/* eslint-disable */
+function removeBook(i) {
+  storeData.splice(i, 1);
+  updateData();
+  displayBooks();
+}
+/* eslint-enable */
+window.onload = displayBooks();
```

Comment:
- [ ] Please do not disable linters as they are meant to help you write readable, maintainable and more consistent code while adhering to best practices 👍

Reply:
Hi, many thanks for pointing out this issue to be resolved.

will it be appropriate to use 

/* eslint-disable no-unused-vars */

Which does not disable linters entirely but indicates to ESLint that the removeBook function is being used in the dynamically generated HTML. This should prevent the ESLint error from appearing.

Reply:
@codepantha